src/Arcus.Scripting.Tests.Integration/Arcus.Scripting.IntegrationAccount.tests.ps1

```diff
@@ -1,5 +1,13 @@
 Import-Module -Name $PSScriptRoot\..\Arcus.Scripting.IntegrationAccount -ErrorAction Stop
 
+function global:New-PartnerFile () {
+    $partnerName = "Partner-$([System.Guid]::NewGuid())"
+    $path = "$PSScriptRoot\Files\IntegrationAccount\Partners\$($partnerName).json"
+    $contents = "{ ""name"": ""$($partnerName)"", ""properties"": { ""partnerType"": ""B2B"", ""content"": { ""b2b"": { ""businessIdentities"": [{ ""qualifier"": ""1"", ""value"": ""12345"" }, { ""qualifier"": ""1"", ""value"": ""54321""} ]} } }}"
```

Comment:
Made this a one-liner as to not ruin dev experience when working on tests, and bc only the partner name is important here.